.github/workflows/check_release_name_bump.yml

```diff
@@ -0,0 +1,41 @@
+name: Check release bump
+
+env:
+  TZ: Europe/London
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: get release-version
+      run: |
+        echo "RELEASE_VERSION=$(printf '%s\n' $(cat .release-version))" >> $GITHUB_ENV
+    
+    - name: get latest-release
+      id: last_release
+      uses: InsonusK/get-latest-release@v1.0.1
```

Comment:
The action claims to be different by providing access to draft releases by we are not really interested in them. Are there perhaps more popular or GitHub created actions to retrieve the releases?

Reply:
I will look around see if there is different, more popular action